.eslintrc.js

```diff
@@ -28,5 +28,11 @@ module.exports = {
      * https://eslint.org/docs/rules/func-names
      */
     'func-names': 'off',
+
+    /**
+     * Disallow enforcement of consistent linebreak style
+     * https://eslint.org/docs/rules/func-names
+     */
+    'linebreak-style': 0,
```

Comment:
Just a very tiny nitpick. For consistency, can you make it "off" instead of 0?

Reply:
Thanks for pointing that out. I totally missed that.